### ISSUE TITLE
2022.11.0 for macOS is unusable

### ISSUE THREAD

**Issue by TallTed** [jitsi/jitsi-meet-electron#810]
## Description
---

Click a past meeting.  Watch the throbber for a second or two, then watch it freeze for 15+ seconds, then get the "Join Meeting".  Click it, and see the gallery for a second or two, with a chat overlay or three ... and wait 15+ seconds for a redraw.  Repeat ad nauseum.

_(*THIS* is why forcing an app update upon launch of an older version is a BAD IDEA.)_

## Current behavior
---

Click a past meeting.  Watch the throbber for a second or two, then watch it freeze for 15+ seconds, then get the "Join Meeting".  Click it, and see the gallery for a second or two, with a chat overlay or three ... and wait 15+ seconds for a redraw.  Repeat ad nauseum.

## Expected Behavior
---

No hangs.

## Possible Solution
---

No idea.

## Steps to reproduce
---

Click a past meeting.  Watch the throbber for a second or two, then watch it freeze for 15+ seconds, then get the "Join Meeting".  Click it, and see the gallery for a second or two, with a chat overlay or three ... and wait 15+ seconds for a redraw.  Repeat ad nauseum.

# Environment details
---

macOS 10.13.6
MacBook Pro (Retina, 15-inch, Mid 2015)
what more do you want?

**Comment by saghul** [comment]
I could not reproduce this, but I'm on a more recent macOS version.

Please share the output when running the app from the terminal like so: `/Applications/Jitsi\ Meet.app/Contents/MacOS/Jitsi\ Meet`

**Comment by TallTed** [comment]
```shell
$ /Applications/Jitsi\ Meet.app/Contents/MacOS/Jitsi\ Meet
11:33:53.963 › Checking for update
11:33:55.779 › Update for version 2022.11.0 is not available (latest version: 2022.11.0, downgrade is disallowed).
11:33:55.780 › checkForUpdatesAndNotify called, downloadPromise is null
```


Immediately after posting this issue, other things (which hadn't shown any problems for the past few days since last-boot) started misbehaving (on a machine which has been stable for months) with similar hangs, which led me to reboot ...

And now everything seems to be behaving properly... 

I don't know if Jitsi exposed an issue elsewhere, or if sunspots got me, or what...

But it doesn't now appear to be Jitsi's fault.  So I'll close this, and reopen (or recreate) if it recurs.
